docs/content/concepts/sui-move-concepts/move-on-sui.mdx

```diff
@@ -28,33 +28,30 @@ Key differences with Move on Sui include:
 
 ### Object-centric global storage {#global-storage}
 
-In Diem-style Move, global storage is part of the programming model that you can access through special operations, such as `move_to`, `move_from`, and more global storage operators. Core Move stores both resources and modules in global storage. When you publish a module, it's stored into a newly generated module address inside Move. When you create a new object (resource), it's usually stored into some address, as well. This approach creates a problem, however, as on-chain storage is expensive and limited (not optimized for storage and indexing). Current blockchains cannot scale effectively to handle storage-heavy applications, such as marketplaces and social apps.
-
-Move on Sui addresses the scaling issue by not having global storage. Consequently, Move doesn't allow any of the global storage-related operations (there is a bytecode verifier to detect violations for this). Instead, storage happens exclusively within Sui. When you publish a module, the newly published module is stored in Sui storage, instead of Move storage. Similarly, newly created objects are stored in Sui storage.
-
-:::info
+In Move on Diem, global storage is part of the programming model. Resources and modules are held in global storage, owned by an account which has an address. Transactions that access resources are only required to specify the types of resources being accessed, and not the accounts they are being accessed from, which can be specified dynamically, using special operations such as `move_to` and `move_from`.
```

Comment:
> Transactions that access resources are only required to specify the types of resources being accessed, and not the accounts they are being accessed from

This was sort of true at a time in Diem, but also not really.

Move on Diem had no fundamental requirements for any types of accesses. There were some safety considerations in modules, but from a transaction standpoint, anything could be accessed.

Reply:
Ah, so the `acquires` annotations on functions were not used for transaction scheduling?